content_script.js

```diff
@@ -17,6 +17,27 @@ class HTMLSerializer {
     this.FILTERED_TAGS = new Set(['script', 'noscript', 'style', 'link']);
 
     /**
+     * @private {Set<string>} Contains the tag names for elements
+     *     that have no closing tags.
```

Comment:
Can you add a link to where you got these from?

The HTML spec is slightly different (https://html.spec.whatwg.org/multipage/syntax.html#void-elements):
area, base, br, col, embed, hr, img, input, keygen, link, meta, param, source, track, wbr

I'd just match whatever chrome does over what the spec says.